ci/lint.yml

```diff
@@ -5,8 +5,8 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: golangci/golangci-lint
-    tag: v1.61.0
+    repository: golang
```

Comment:
Is changing from 'golangci/golangci-lint' to 'golang' is correct
golangci/golangci-lint  image build instructions are here:
https://hub.docker.com/layers/golangci/golangci-lint/v1.62.2-alpine/images/sha256-6f5514ee6ae7b36e3028d5583036b3551a289e2f621edc1f6d9b35af33c4ed2b

and it ends up running the command: 
CMD ["golangci-lint"]

Reply:
have you tried this locally and introduced some linting errors to see if you get what you expect ?

Reply:
I'm following what other repos have := https://github.com/ONSdigital/dp-frontend-release-calendar/blob/develop/ci/lint.yml

...and yes I have tested it